akka-docs/rst/scala/code/docs/akka/typed/IntroSpec.scala

```diff
@@ -66,6 +66,28 @@ object IntroSpec {
         }
       }
     //#chatroom-behavior
+
+    //#chatroom-actor
+    def mutableChatRoom(): Behavior[Command] =
+      Deferred[Command] { _ =>
+        var sessions: List[ActorRef[SessionEvent]] = List.empty
+        Stateful[Command] { (ctx, msg) ⇒
```

Comment:
@rkuhn Let me throw some paint on that shed also. As mentioned in #22325 I want to remove `stateless`. Actors are by definition sateful, with some rare exceptions. Symmetry is nice and therefore I suggest that we rename `stateful` to `immutable`. Then we have two (and only two) different styles, and they are so different that they deserve different APIs.